src/helpers/debugShallow.ts

```diff
@@ -1,20 +1,22 @@
-// @flow
 import * as React from 'react';
+import type { ReactTestInstance } from 'react-test-renderer';
 import { shallowInternal } from '../shallow';
 import format from './format';
 
 /**
  * Log pretty-printed shallow test component instance
  */
 export default function debugShallow(
-  instance: ReactTestInstance | React.Element<any>,
+  instance: ReactTestInstance | React.ReactElement<any>,
   message?: string
 ) {
   const { output } = shallowInternal(instance);
 
   if (message) {
+    // eslint-disable-next-line no-console
```

Comment:
Since were library that sometimes need to communicate with user, maybe we could disable `no-console` rule in ESLint config. @thymikee wdyt?

Reply:
Nah, I'd rather create a dummy logger helper and disable the rule in one place. It's still possible to leave some debug code using console.logs which would be super annoying if happens